kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -452,6 +455,7 @@ export function showExam(store, classId, examId, questionNumber) {
         } else {
           const contentPromise = ContentNodeResource.getCollection({
             ids: questionSources.map(item => item.exercise_id),
+            by_role: true,
```

Comment:
I think we can skip it here, as if a coach has explicitly added it to the exam, we want to display this to the end user.